src/nifti/parseNiftiFile.js

```diff
@@ -1,6 +1,7 @@
 import { external } from '../externalModules.js';
 import decodeNiFTIBigEndian from '../shared/niftiBigEndianDecoder.js';
 import normalizeInvalid from '../shared/normalizeInvalid.js';
+import _ from 'lodash';
```

Comment:
Lodash needs to be declared as a `dependency` rather than a `devDependency` in package.json in order to use it in src. When used here also please import specific functions to cut down on dist size.

```js
import cloneDeep from 'lodash/cloneDeep';
```

Reply:
Removed the import and affine matrix copying done with Javascript array functions.